climada/util/plot.py

```diff
@@ -272,6 +272,9 @@ def geo_im_from_array(array_sub, coord, var_name, title,
         coordinate reference system used in coordinates, by default None
     smooth : bool, optional
         smooth plot to RESOLUTIONxRESOLUTION, by default True
+    shapes : bool, optional
+        Overlay Earth's countries coastlines to matplotlib.pyplot axis.
+        The default is True
```

Comment:
I would call this `coastlines` instead of `shapes`. Much clearer.

Reply:
Since other functioms like `geo_bin_from_array()`,  `geo_scatter_from_array`, etc, they all name them as shapes for turning on/off the plotting of coastline, I would suggest either having it as `shapes` in `geo_im_from_array`, or renaming others as `coastlines`, to make it more consistent.

Reply:
... and for the sake of backwards compatibility I'd opt for `shapes`

Reply:
Ok, let it be `shapes` then. It is a very bad name, but consistency of code is probably more important.